test/Microsoft.DotNet.Tools.Builder.Tests/IncrementalTestBase.cs

```diff
@@ -13,29 +13,29 @@ namespace Microsoft.DotNet.Tools.Builder.Tests
 {
     public class IncrementalTestBase : TestBase
     {
-        protected readonly TempDirectory _tempProjectRoot;
+        protected readonly TempDirectory TempProjectRoot;
 
-        private readonly string _testProjectsRoot;
-        protected readonly string _mainProject;
-        protected readonly string _expectedOutput;
+        protected readonly string TestProjectsRoot;
```

Comment:
Is this used anywhere else outside of the constructor? If not, you can get rid of it.